### PR TITLE
create commandsist for device

### PR DESCRIPTION
**Pull request by StimkaT**

### CODE REVIEW

File: projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts [hunk 1]
```diff
@@ -4,9 +4,15 @@ export const BLINK_COMMANDS_LIST_FEATURE_KEY = 'blink/commandsList';
 
 export interface IBlinkCommands {
     name: string;
-    event: string;
+    description:string;
+    event: any;
 }
 
+// export interface IEvent {
```
Comment (StimkaT) [inline projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts:11]: remove comments

File: projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts [hunk 2]
```diff
@@ -16,9 +22,9 @@ export interface BlinkCommandsListPartialState {
 }
 
 export const initialState: BlinkCommandsListState = {
-    commands: [ {name: 'ON Indicator', event: '{"event":"LED","data":{"command":"ON"}}'},
```
Comment (StimkaT) [inline projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts:19]: space and transfer object to next string

File: projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts [hunk 3]
```diff
@@ -16,9 +22,9 @@ export interface BlinkCommandsListPartialState {
 }
 
 export const initialState: BlinkCommandsListState = {
-    commands: [ {name: 'ON Indicator', event: '{"event":"LED","data":{"command":"ON"}}'}, 
-    {name: 'OFF Indicator', event: '{"event":"LED","data":{"command":"OFF"}}'},
-    {name: 'BLINK Indicator', event: '{"event":"LED","data":{"command":"BLINK"}}'}]
+    commands: [ {name: 'ON Indicator', description:'{event:"LED",data:{command:"ON"}}', event: {event:"LED",data:{command:"ON"}}} ,
```
Comment (StimkaT) [inline projects/blink/src/lib/+state/blink-commands-list/blink-commands-list.reducer.ts:25]: space and transfer object to next string

File: projects/blink/src/lib/containers/blink-commands-list-container/blink-commands-list-container.component.ts [hunk 4]
```diff
@@ -15,8 +17,21 @@ import { AsyncPipe } from '@angular/common';
   styleUrl: './blink-commands-list-container.component.scss'
 })
 export class BlinkCommandsListContainer {
+
   commands$ = this.store$.select(getBlinkCommandsList);
 
-  constructor(private store$: Store) {
+  constructor(private store$: Store,
```
Comment (StimkaT) [inline projects/blink/src/lib/containers/blink-commands-list-container/blink-commands-list-container.component.ts:23]: transfer to new line

File: projects/blink/src/lib/containers/blink-three-container/blink-three-container.component.html [hunk 5]
```diff
@@ -1,6 +1,6 @@
 <page-layout>
   <blink-nav-panel-container toplBar></blink-nav-panel-container>
   <div body>
-    <light-three (emitter)="events($event)"></light-three>
+    <light-three (emitter)="events($event)" ></light-three>
```
Comment (StimkaT) [inline projects/blink/src/lib/containers/blink-three-container/blink-three-container.component.html:4]: space

File: projects/blink/src/lib/containers/blink-nav-panel-container/blink-nav-panel-container.component.ts [hunk 6]
```diff
@@ -26,7 +26,7 @@ export class BlinkNavPanelContainerComponent {
     } else  if ($event.event === 'NavDeviceComponent:BUTTON_CLICKED' && $event.data === 'aboutDWidget') {
       this.router.navigate(['control/blink/:deviceName/aboutDevWidget']);
     } else  if ($event.event === 'NavDeviceComponent:BUTTON_CLICKED' && $event.data === 'controlDevice') {
-      this.router.navigate(['control/blink/:deviceName']);
+      this.router.navigate(['control/blink/**']);
```
Comment (sislex) [inline projects/blink/src/lib/containers/blink-nav-panel-container/blink-nav-panel-container.component.ts:29]: this.router.navigate(['control/blink/:deviceName']); is better now

**Pull request merged**